app/pages/error/contextCreator.test.js

```diff
@@ -0,0 +1,82 @@
+    expect(context.error.title).toEqual(defaultContext.error.title);
+    expect(context.error.description).toEqual(defaultContext.error.description);
+  });
+
+  it('should get the error status with the default error context', () => {
+    const context = getContext({
+      status: 999,
+    });
+
+    expect(context.error.status).toEqual(context.error.status);
+    expect(context.error.backLinkText).toEqual(defaultContext.error.backLinkText);
+    expect(context.error.backLinkHref).toEqual(defaultContext.error.backLinkHref);
+    expect(context.error.title).toEqual(defaultContext.error.title);
+    expect(context.error.description).toEqual(defaultContext.error.description);
+  });
+
+  it('should get the error backLinkText with the default error context', () => {
+    const context = getContext({
+      backLinkText: 'Error backLinkText',
+    });
+
+    expect(context.error.status).toEqual(defaultContext.error.status);
+    expect(context.error.backLinkText).toEqual(context.error.backLinkText);
+    expect(context.error.backLinkHref).toEqual(defaultContext.error.backLinkHref);
+    expect(context.error.title).toEqual(defaultContext.error.title);
+    expect(context.error.description).toEqual(defaultContext.error.description);
+  });
+
+  it('should get the error backLinkHref with the default error context', () => {
+    const context = getContext({
+      backLinkHref: 'http://errorBackLinkHref.com',
+    });
+
+    expect(context.error.status).toEqual(defaultContext.error.status);
+    expect(context.error.backLinkText).toEqual(defaultContext.error.backLinkText);
+    expect(context.error.backLinkHref).toEqual(context.error.backLinkHref);
+    expect(context.error.title).toEqual(defaultContext.error.title);
+    expect(context.error.description).toEqual(defaultContext.error.description);
+  });
+
+  it('should get the error title with the default error context', () => {
+    const context = getContext({
+      title: 'Error Title',
+    });
+
+    expect(context.error.status).toEqual(defaultContext.error.status);
+    expect(context.error.backLinkText).toEqual(defaultContext.error.backLinkText);
+    expect(context.error.backLinkHref).toEqual(defaultContext.error.backLinkHref);
+    expect(context.error.title).toEqual(context.error.title);
+    expect(context.error.description).toEqual(defaultContext.error.description);
+  });
+
+  it('should get the error title with the default error context', () => {
+    const context = getContext({
+      description: 'Error Description',
+    });
+
+    expect(context.error.status).toEqual(defaultContext.error.status);
+    expect(context.error.backLinkText).toEqual(defaultContext.error.backLinkText);
+    expect(context.error.backLinkHref).toEqual(defaultContext.error.backLinkHref);
+    expect(context.error.title).toEqual(defaultContext.error.title);
+    expect(context.error.description).toEqual(context.error.description);
+  });
```

Comment:
Nice but I would just assert the single thing you are testing for. So here just the description and the rest can be removed. The first test ensures all the defaults are checked